tests/test_experiment.py

```diff
@@ -229,3 +235,77 @@ def test_telemetry_default(
 def test_error_on_cobalt() -> None:
     with pytest.raises(SSUnsupportedError):
         exp = Experiment("cobalt_exp", launcher="cobalt")
+
+
+def test_default_orch_path(test_dir: str, wlmutils: "conftest.WLMUtils") -> None:
+    """Ensure the default file structure is created for Orchestrator"""
+
+    exp_name = "default-orch-path"
+    exp = Experiment(exp_name, launcher="local", exp_path=test_dir)
+    db = exp.create_database(
+        port=wlmutils.get_test_port(), interface=wlmutils.get_test_interface()
+    )
+    exp.start(db)
+    exp.stop(db)
+    orch_path = pathlib.Path(test_dir) / db.name
+    err_file = os.path.join(orch_path, f"{db.name}_0.err")
+    out_file = os.path.join(orch_path, f"{db.name}_0.out")
+    assert orch_path.exists()
+    assert os.path.isfile(err_file)
+    assert os.path.isfile(out_file)
+    assert db.path == str(orch_path)
+
+
+def test_default_model_path(test_dir: str) -> None:
+    """Ensure the default file structure is created for Model"""
+
+    exp_name = "default-model-path"
+    exp = Experiment(exp_name, launcher="local", exp_path=test_dir)
+    settings = exp.create_run_settings(exe="echo", exe_args="hello")
+    model = exp.create_model(name="nerp", run_settings=settings)
+    exp.start(model)
```

Comment:
Same as before, instead of actually starting the model, monkeypatch the method so that it doesn't actually execute the model